clusterloader2/docs/test_suite_design.md

```diff
@@ -0,0 +1,46 @@
+# Cluster loader - TestSuite
+
+## Background
+
+Since 08/2018 cluster loader become main tool for kubernetes e2e performance tests.
+It's is a very convenient tool for running single test case, but not for running
+test bundles. Currently it is impossible to run the same test
+with two different parameters sets in a single job run.
+This forces developers to create separate job configs for
+almost each tests.
+
+## TestSuite
+
+TestSuite is a test bundle.
+Overrides is a set of override file paths. Overrides provided on the TestSuite level will be applied
+to all of the test, unless they are overidden on the test level.
```

Comment:
How they can be overridden? Is it decided individually for every single env-var?